<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="general_groupBox">
     <property name="title">
      <string>General Preferences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="general_formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="addNewMarkersToLabel">
          <property name="text">
           <string>Add New Markers to:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="addNewMarkersToComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="addNewLinesToLabel">
          <property name="text">
           <string>Add New Lines to:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="addNewLinesToComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="solverUI_groupBox">
     <property name="title">
      <string>Solver UI Preferences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="solverUI_formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="minimalUIWhileSolvingLabel">
          <property name="text">
           <string>Minimal UI While Solving:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="minimalUIWhileSolvingComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="solverOptionsLabel">
          <property name="text">
           <string>Solver Options</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="solverOptionsComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="loadMarkerUI_groupBox">
     <property name="title">
      <string>Load Marker UI Preferences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QFormLayout" name="loadMarkerUI_formLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="useOverscanDefaultLabel">
          <property name="text">
           <string>Use Embedded Overscan Default:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="useOverscanDefaultComboBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="loadBundlePositionsDefaultLabel">
          <property name="text">
           <string>Load Bundle Positions Default:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="loadBundlePositionsDefaultComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="distortionModeDefaultLabel">
          <property name="text">
           <string>Distortion Mode Default:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="distortionModeDefaultComboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="renameMarkersDefaultLabel">
          <property name="text">
           <string>Rename Markers Default:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="renameMarkersDefaultComboBox"/>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="bundleSpaceDefaultComboBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="bundleSpaceDefaultLabel">
          <property name="text">
           <string>Bundle Space Default:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="deprecated_groupBox">
     <property name="title">
      <string>Deprecated Preferences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QFormLayout" name="deprecated_formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="validateSolverOnUIOpenLabel">
          <property name="text">
           <string>Validate on open:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="validateSolverOnUIOpenComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="showValidateButtonLabel">
          <property name="text">
           <string>Show Validate Button:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="showValidateButtonComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="allowObjectRelationsLabel">
          <property name="text">
           <string>Allow Object Relationships</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="allowObjectRelationsComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
